native/package.json

```diff
@@ -61,7 +61,6 @@
     "react-native-floating-action": "^1.19.1",
     "react-native-fs": "2.15.2",
     "react-native-gesture-handler": "^1.6.1",
-    "react-native-hyperlink": "git+https://git@github.com/ashoat/react-native-hyperlink.git#both",
```

Comment:
One more thing I realized - we should `git rm patches/react-native-hyperlink+0.0.12.patch`